dispatch/static/manager/package.json

```diff
@@ -54,5 +55,8 @@
   },
   "bugs": {
     "url": "https://github.com/ubyssey/dispatch/issues"
+  },
+  "dependencies": {
+    "moment": "^2.17.1"
```

Comment:
hmm not sure how it got into devDependencies since i ran npm with the --save tag. I didn't realize our dependencies were all devDependencies, is there any reason why that is? I believe the convention is to save anything required for our app to function as a dependency so when we install it in a production env (with the --production flag for npm) it'll ignore the devDependencies, which are generally linters, test runners, etc. 

I will remove moment from dependencies and leave the one in the devDependency object, since that is the convention this project is using.